<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JA_ProjektClass</class>
 <widget class="QMainWindow" name="JA_ProjektClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>916</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>JA_Projekt</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>131</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="LoadImageButton">
       <property name="text">
        <string>Load image</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="CppRadioButton">
       <property name="text">
        <string>C++</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="AsmRadioButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>ASM</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="FilterOnButton">
       <property name="text">
        <string>Filter On</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="HistogramButton">
       <property name="text">
        <string>Input Histogram</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="OutputHistogramButton">
       <property name="text">
        <string>Output Histogram</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="SaveImageButton">
       <property name="text">
        <string>Save image</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>731</width>
      <height>431</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout"/>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>131</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="MessageText">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>239</y>
      <width>131</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="ExecutionText">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="JA_Projekt.qrc"/>
 </resources>
 <connections/>
</ui>
